sdk/core/azure-core/src/main/java/com/azure/core/implementation/tracing/TracerProxy.java

```diff
@@ -38,6 +38,26 @@ public static Context start(String methodName, Context context) {
         return local;
     }
 
+    /**
```

Comment:
Still not a fan of TracerProxy being a static class. makes it hard to test

Reply:
Can we instead, pass a "Tracer" into EventHubClient as a ctor parameter and propagate that? That way, we can mock the other calls.